soccer/Robot.cpp

```diff
@@ -67,6 +67,7 @@ OurRobot::OurRobot(int shell, SystemState* state)
     //_lastChargedTime = 0;
     _lastKickerStatus = 0;
     //_lastKickTime = 0;
+    _lastBallSense = RJ::Time();
```

Comment:
Change this to zero in case it causes some random bug when all robots sense the ball at the start.

Reply:
RJ::Time() is 0, it is not the same as RJ::now()